src/test/java/BoardTest.java

```diff
@@ -0,0 +1,159 @@
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BoardTest {
+
+    Board board = new Board();
+
+    @Test
+    public void getSquareValueReturnsEmptyWhenSquareEmpty() {
+       assertEquals(Player.NEITHER, board.getSquareValue(0, 0));
+    }
+
+    @Test
+    public void getSquareValueReturnsPlayer1WhenSquareTakenByPlayerX() {
+        board.setSquareValue(0, 0, Player.X);
+        assertEquals(Player.X, board.getSquareValue(0, 0));
+    }
+
+    @Test
+    public void getSquareValueReturnsPlayer2WhenSquareTakenByPlayerO() {
+        board.setSquareValue(0, 0, Player.O);
+        assertEquals(Player.O, board.getSquareValue(0, 0));
+    }
+
+    @Test
+    public void isEmptySquareReturnsTrueWhenSquareIsEmpty() {
```

Comment:
Try not to encode the mechanics in your test. emptySquareIsEmpty is what I'd call this one. Smell to me: the use of Returns in a test name.